src/screens/Customers/AddMeterReading.tsx

```diff
@@ -1,61 +1,101 @@
 import { Typography } from '@material-ui/core';
 import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
-import React from 'react';
-import { RouteComponentProps } from 'react-router-dom';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RouteComponentProps, Redirect, useHistory } from 'react-router-dom';
 import BaseScreen from '../../components/BaseComponents/BaseScreen';
 import Button from '../../components/Button';
 import TextField from '../../components/TextField';
+import { CustomerRecord } from '../../lib/airtable/interface';
+import { selectCurrentCustomer } from '../../lib/redux/customerData';
+import { EMPTY_METER_READING } from '../../lib/redux/customerDataSlice';
+import moment from 'moment';
+import { selectCurrentUserId } from '../../lib/redux/userData';
+import { getTariffPlanByCustomer, calculateAmountBilled } from '../../lib/utils/customerUtils';
+import { createMeterReadingAndUpdateCustomerBalance } from '../../lib/airtable/request';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { formatDateStringToLocal } from '../../lib/moment/momentUtils';
+
 
 const styles = (theme: Theme) =>
   createStyles({
-    content: {
-      color: theme.palette.text.primary,
-    },
-    outlined: {
-      padding: '15px 15px 10px',
-      border: `1px solid ${theme.palette.text.secondary}`,
-      borderRadius: '6px',
-      marginTop: '10px',
-    },
-    secondaryText: {
-      color: theme.palette.text.secondary,
+    amountOwedContainer: {
+      margin: '20px 0px'
```

Comment:
Nit: not a big deal at all but it's nice to use `theme.spacing(x)` values when possible to keep our spacing standardized around multiples of 8.

e.g. `margin: theme.spacing(3, 0)` is equivalent to 24px 0px. `theme.spacing(2)` is 16px etc.